### DISCUSSION TITLE
[Request] autocomplete users

### DISCUSSION THREAD

**Discussion by xylolink** [hyperspacedev/hyperspace#182] [Ideas]
**Is your feature request related to a problem? Please describe.**
need autocompletion of mentionned users when starting a new post

**Describe the solution you'd like**
Typing "@" automatically suggest accounts from followers/following list
Typing any other caracters filters the account list with these caracters

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by alicerunsonfedora** [comment]
We can definitely take a look at implementing something like this in a future feature update after v1.1.0.
